### PR TITLE
{lib}[intel/2017b] FoX v4.1.2

### PR DESCRIPTION
**Pull request by Micket**
New module which is a dependency for newest version of QuantumESPRESSO (starting 6.2).

### CODE REVIEW

File: easybuild/easyconfigs/f/FoX/FoX-4.1.2-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': False, 'openmp': False}
+
+sources = [
+    SOURCE_TAR_GZ,
+]
+
+source_urls = [
+    'http://homepages.see.leeds.ac.uk/~earawa/FoX/source/'
+]
```
Comment (boegel) [inline easybuild/easyconfigs/f/FoX/FoX-4.1.2-intel-2017b.eb]: @Micket Please include a SHA256 checksum, and collapse re-order `source_urls` and `sources`:

```
sources = ['...']
sources = [SOURCE_TAR_GZ]
checksums = ['...']
```

(`eb --inject-checksums FoX-4.1.2-intel-2017b.eb`) will do that for you.

File: easybuild/easyconfigs/f/FoX/FoX-4.1.2-intel-2017b.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+name = 'FoX'
+version = '4.1.2'
+
+homepage = 'http://homepages.see.leeds.ac.uk/~earawa/FoX/'
+description = """FoX is an XML library written in Fortran 95.
+It allows software developers to read, write and modify XML documents from Fortran applications without the
+complications of dealing with multi-language development."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': False, 'openmp': False}
```
Comment (boegel) [inline easybuild/easyconfigs/f/FoX/FoX-4.1.2-intel-2017b.eb]: these are both disabled by default, so this line can be removed?

**Comment by Micket** [comment]
@boegel Done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2603.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/30a544afba3d65fa8b8285ca85581cd7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2011.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/90ceb810e24ac6133fa02ac4b2bbc57a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**